app/notifications/utils.py

```diff
@@ -1,10 +1,10 @@
 import requests
 from flask import current_app
 
+from app.constants import HTTP_TIMEOUT
 
-def confirm_subscription(confirmation_request):
-    from app import HTTP_TIMEOUT  # Circular import
 
+def confirm_subscription(confirmation_request):
```

Comment:
Annotation?  It quacks like a dictionary.

Reply:
Letter code, not updating.